### ISSUE TITLE
Insufficient color contrast of buttons

### ISSUE THREAD

**Issue by limonte** [sweetalert2/sweetalert2#2354]
## Current behavior

- OK button - contrast ratio 4.26: https://app.contrast-finder.org/result.html?foreground=%23FFF&background=%237367F0&ratio=4.5&isBackgroundTested=true&algo=Rgb&lang=en
- Cancel button - 3.56: https://app.contrast-finder.org/result.html?foreground=%23FFF&background=%23ea5455&ratio=4.5&isBackgroundTested=true&algo=Rgb&lang=en
- Deny button - 4.24: https://app.contrast-finder.org/result.html?foreground=%23FFF&background=%236e7d88&ratio=4.5&isBackgroundTested=true&algo=Rgb&lang=en

## Expected behavior

Color contrast for UI components should be at least 4.5
